### PR TITLE
DOCS-5584 acm-lightning-components

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Based on review in https://github.com/mulesoft/docs-api-community-manager/pull/50

**Comment by wwarner-salesforce** [comment]
@kbostrom , please only review the edits to these files, and not the unedited parts of the files which will be covered later.

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
  ** xref:install-validate.adoc[Verify the Installation]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc:7]: NOTE:
Very quick copyedit of changed material only.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 2]
```diff
@@ -1,270 +1,306 @@
-= ACM Lightning Components
+= Description of ACM Lightning Components
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-To design your API Community Manager portal, open the Builder and add ACM Lightning Components as described in xref:customize.adoc[Customize the Look and Feel of Your API Community].
+The pages of your API Community Manager (ACM) portal are designed by adding and configuring Lightning Components in the Builder.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Reword to use active voice.

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Also, for this and all uses of "Lightning" in this doc, follow the Lightning Components Style Guide:

https://docs.google.com/document/d/1UL9a6cVZLu0lajgWOYiD_tHM8qp4_JRzFjCfGLwv01g/edit

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 3]
```diff
@@ -1,270 +1,306 @@
...
 
-This list explains each of the ACM Lightning Components, the functionality they add to the portal, and their configuration options.
+These Lightning Components include both the components available in all Salesforce Communities, and the ACM Lightning Components only available in ACM.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
These Lightning components include both the components available in all Salesforce Communities, and the ACM Lightning components available only in ACM.
```

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 4]
```diff
@@ -1,270 +1,306 @@
...
+
+Each ACM Lightning Component has different configuration options and adds different functionality to the portal.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
Each ACM Lightning component has different configuration options and adds different functionality to the portal.
```

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: I ran a search for a "Builder" but did not have much luck. Either this is not the complete official name, or we should lowercase it to "builder."

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 5]
```diff
@@ -1,270 +1,306 @@
...
 
 == API Access Requester
 
-The API access requester button component enables members to request access to an API for an application.
+The *API Access Requester* component enables community members to request access to an API for either a new or existing application.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: GLOBALLY, I recommend that we NOT boldface component names. Init. cap. (with lowercase "component") should suffice.
I will stop marking; please implement throughout.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 6]
```diff
@@ -1,270 +1,306 @@
...
-Show Buttons:: Show the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
-Card Height:: Set the card height.
-Card Margin:: Set the amount of margin space around the card.
-Card Border:: Set the thickness of the card borders.
-Card Border Radius:: Set the radius of the card borders. A larger radius makes the cards look more rounded.
-Card Shadow:: Set the shadow beneath the card.
-API Details Button Label:: Set the label of the *API Details* button. Default is `API Details`.
-Learn More Button Label:: Set the label of the *Learn More* button. Default is `Learn More`.
+API:: Specifies the API shown.
+Override URL:: If set, overrides the default *API Details* button URL to the API main page in favor of linking to other information that you want the user to see first, such as documentation or marketing collateral.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: What is a button URL?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 7]
```diff
@@ -1,270 +1,306 @@
...
+Show Buttons:: Shows the *API Details* button. If the button is not shown, users can click anywhere on the card to show the API details.
+Card Height:: Specifies the card height.
+Card Margin:: Specifies the amount of margin space around the card.
+Card Border:: Specifies the card border thickness.
+Card Border Radius:: Specifies the card border radius. A larger radius makes the cards look more rounded.
+Card Shadow:: Specifies the shadow beneath the card.
+API Details Button Label:: Enables you to set the *API Details* button label. Default is `API Details`.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
API Details Button Label:: Enables you to set the *API Details* button label. Default is *API Details*.
```
Do not use monospace to portray UI labeling. GLOBAL; I will stop marking from here.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 8]
```diff
@@ -1,270 +1,306 @@
...
+You can specify values for the following parameters of the *API Console*:
+
+API:: Specifies the API to display.
+Show Documentation Pages:: Shows the API's documentation pages.
+Link Hover Color:: Specifies the hover link color.
+Button Text Color (Optional):: Specifies the color of button text.
+Button Hover Text Color (Optional):: Specifies the color of button hover text. Default is the button text color.
+Headline Font Family (Optional):: Specifies the API console headline font family.
+Error Color (Optional):: Specifies the color of error messages.
+Method Label Get Background Color:: Specifies the *GET* method label background color.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: GLOBALLY unbold GET, POST, PUT, DELETE, and PATCH.

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: GLOBALLY consider whether we really need to document UI elements that are so intuitively named.
Even a very nontechnical person like me can understand the labels without the descriptions.
This is a larger question that applies to a lot of our doc.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 9]
```diff
@@ -1,270 +1,306 @@
...
-Method Label Delete Background Color:: Set the background color of the labels of *DELETE* methods.
-Method Label Delete Color:: Set the color of the labels of *DELETE* methods.
-Method Label Patch Background Color:: Set the background color of the labels of *PATCH* methods.
-Method Label Patch Color:: Set the color of the labels of *PATCH* methods.
-Method Label Options Background Color:: Set the background color of the labels of *OPTIONS* methods.
-Method Label Options Color:: Set the color of the labels of *OPTIONS* methods.
-Method Label Head Background Color:: Set the background color of the labels of *HEAD* methods.
-Method Label Head Color:: Set the color of the labels of *HEAD* methods.
-Headline Font Family (Optional):: Set the font family of the API console headline.
+The *API Console Documentation* component shows the documentation from Anypoint Exchange about an API.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
The *API Console Documentation* component shows the Anypoint Exchange documentation about an API.
```

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 10]
```diff
@@ -1,270 +1,306 @@
...
+
+You can specify values for the following parameters of the *API Console Documentation*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Ditto previous edits about GET etc. and whether we need this list.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 11]
```diff
@@ -1,270 +1,306 @@
...
 
 == API Console Instances
 
-The API console instances component shows the list of instances of an API.
+The *API Console Instances* component shows the list of instances of an API.
 
 // check component description
 
-API:: Choose the API whose instances are displayed.
+You can specify values for the following parameters of the *API Console Instances*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Why use a list for just one item?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 12]
```diff
@@ -1,270 +1,306 @@
...
-Method Label Put Color:: Set the color of the labels of *PUT* methods.
-Method Label Delete Background Color:: Set the background color of the labels of *DELETE* methods.
-Method Label Delete Color:: Set the color of the labels of *DELETE* methods.
-Method Label Patch Background Color:: Set the background color of the labels of *PATCH* methods.
-Method Label Patch Color:: Set the color of the labels of *PATCH* methods.
-Method Label Options Background Color:: Set the background color of the labels of *OPTIONS* methods.
-Method Label Options Color:: Set the color of the labels of *OPTIONS* methods.
-Method Label Head Background Color:: Set the background color of the labels of *HEAD* methods.
-Method Label Head Color:: Set the color of the labels of *HEAD* methods.
+You can specify values for the following parameters of the *API Console Navigation*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto prev. list edits

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 13]
```diff
@@ -1,270 +1,306 @@
...
-Method Label Head Background Color:: Set the background color of the labels of *HEAD* methods.
-Method Label Head Color:: Set the color of the labels of *HEAD* methods.
-Link Hover Color:: Set the color of links when the cursor hovers over them.
-Button Text Color (Optional):: Set the color of text on buttons.
-Button Hover Text Color (Optional):: Set the color of text on buttons when the cursor hovers over them. Default is the button text color.
-Headline Font Family (Optional):: Set the font family of the headline.
-Error Color (Optional):: Set the color of error messages.
+The *API Console Request Panel* component shows a panel where users can test an API by making requests to it.
+
+You can specify values for the following parameters of the *API Console Request Panel*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 14]
```diff
@@ -1,270 +1,306 @@
...
+Button Hover Text Color (Optional):: Specifies the color of button hover text. Default is the button text color.
+Headline Font Family (Optional):: Specifies the headline font family.
+Error Color (Optional):: Specifies the color of error messages.
 
 == API Details Button
 
-The API details button component provides portal users with details about an API.
+The *API Details Button* component provides portal users with details about an API.
+
+You can specify values for the following parameters of the *API Details Button*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 15]
```diff
@@ -1,270 +1,306 @@
...
+API:: Specifies the API to display.
+Button Label:: Specifies the button label. Default is `API Details`.
+Button Align:: Specifies button alignment as left, right, or center of the UI field.
+API Name:: Adds the API name to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
+API Version:: Adds the API version to the button label. Values are *Append*, *Prepend*, or *None*. Default is *None*.
 
 == API Documentation Viewer
 
-The API documentation viewer component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
+The *API Documentation Viewer* component shows an API's documentation. The documentation may have multiple pages, and each page may have multiple sections.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Do not use "may." See style reference. GLOBAL

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 16]
```diff
@@ -1,270 +1,306 @@
...
+
+You can specify values for the following parameters of the *API Documentation Viewer*:
 
-API:: Choose the API.
-Section (Optional):: Set the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
-Selected Page (Optional):: Set the page to open.
+API:: Specifies the API.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:198]: ditto nec?

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 17]
```diff
@@ -1,270 +1,306 @@
...
+Section (Optional):: Selects the section to open. If the *Selected page (optional)* setting is empty, the *Section (optional)* setting is ignored.
+Selected Page (Optional):: Selects the page to open.
 
 == API Header
 
-The API header component displays the API Header including icon, name, and description.
+The *API Header* component displays the API header, including icon, name, and description.
 
-API:: Choose the API to show.
+You can specify values for the following parameters of the *API Header*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto one item

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 18]
```diff
@@ -1,270 +1,306 @@
...
 
 == Application Analytics Graph
 
-The application analytics graph component displays statistics about the use of an application.
+The *Application Analytics Graph* component displays statistics about the use of an application, such as traffic, response time, and application health.
+
+You can specify values for the following parameters of the *Application Analytics Graph*:
 
-Application:: Choose the application to show.
+Application:: Specifies the application to show.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc:216]: ditto one item

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 19]
```diff
@@ -1,270 +1,306 @@
...
 
 == Application Details
 
-The application details component displays details about an application.
+The *Application Details* component displays details about an application.
 
 // check component description
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Internal comments should not appear in any text that is kept in the public repo. GLOBALLY replace these with Jiras going forward.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 20]
```diff
@@ -1,270 +1,306 @@
...
 
-Application:: Choose the API to show.
+You can specify values for the following parameters of the *Application Details*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 21]
```diff
@@ -1,270 +1,306 @@
...
+The *Application Listing* component shows applications that use an API.
 
-New Application URL:: Set a new URL for the application.
-Card Margin:: Set the amount of margin space around the card.
-Card Border:: Set the thickness of the card borders.
-Card Border Radius:: Set the radius of the card borders. A larger radius makes the cards look more rounded.
-Card Shadow:: Set the shadow beneath the card.
-Card Width:: Set the card width.
-Card Height:: Set the card height.
+You can specify values for the following parameters of the *Application Listing*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 22]
```diff
@@ -1,270 +1,306 @@
...
+
+New Application URL:: Specifies a new application URL.
+Card Margin:: Specifies the amount of margin space around the card.
+Card Border:: Specifies the card border thickness.
+Card Border Radius:: Specifies the card border radius. Cards with a larger radius look more rounded.
+Card Shadow:: Specifies the shadow beneath the card.
+Card Width:: Specifies the card width.
+Card Height:: Specifies the card height.
 
 // check *New Application URL* description
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: Please remove and replace with a Jira. Should be resolved.

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 23]
```diff
@@ -1,270 +1,306 @@
...
 
 == Learn More Button
 
-The learn more button component provides portal users with a custom page with additional information about the API.
+The *Learn More Button* component provides portal users with a custom page with additional information about the API.
+
+You can specify values for the following parameters of the *Learn More Button*:
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]: ditto

Comment (kbostrom) [reply modules/ROOT/pages/acm-lightning-components.adoc]: ENDING EDIT HERE. This is all repetitive and my previous comments apply to all the rest.

**Review by kbostrom — APPROVED** [review]
Here's a second.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 7 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Implemented globally.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Reworded to use active voice, fixed capitalization to follow Lightning Components Style Guide, and changed "Builder" to complete name "Community Builder". Will update instances of Lightning and Builder in other files in a future PR.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Implemented.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: When the user clicks the button they go to the button URL. Rewrote for clarity.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 8 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Unbolded. Agreed, the docs on these properties and the one-item lists need review.

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 15 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Removed "may."

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 19 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Removed to Jiras.

**Pull request merged**